src/Stripe/Entities/StripeTransfer.cs

```diff
@@ -18,7 +18,7 @@ public class StripeTransfer : StripeObject
         public int Amount { get; set; }
 
         [JsonProperty("application_fee")]
-        public int? ApplicationFee { get; set; }
+        public string ApplicationFee { get; set; }
```

Comment:
Added, however it doesn't appear to be expandable according to the documentation
https://stripe.com/docs/api#transfer_object